app/controllers/todos_controller.rb

```diff
@@ -5,13 +5,9 @@ class TodosController < ApplicationController
   expose(:todos) { current_user.todos }
 
   def create
-    current_user.todos.create(todo_params)
-    redirect_to todos_path
+    todo_params = params[:todo]
+    current_user.todos.create(title: todo_params[:title])
+    redirect_to '/'
   end
 
-  private
-
-  def todo_params
-    params.require(:todo).permit(:title)
-  end
```

Comment:
This goes **double** for the ```permit``` method.  It is covered in your tests as you specifically wrote a test against ```title```... but I swear, most Rails devs won't do that.  Or they might test to start, but then they just start hacking properties in-and-out.

I never understood why the white-listing was moved from the model to the parameters itself, it's just... weird.